.circleci/config.yml

```diff
@@ -656,170 +656,63 @@ workflows:
       - schedule:
-          cron: "0 5 * * *"
+          cron: '0 5 * * *'
           filters:
             branches:
               only: /(^master$)/
 
-  # run tests for PRs/branches other than master
-  test-branch:
-    jobs:
-      - test-libs:
-          filters:
-            branches:
-              ignore: /(^master$)|(.*?hotfix(.*)?)/
-      - test-contracts:
-          filters:
-            branches:
-              ignore: /(^master$)|(.*?hotfix(.*)?)/
-      - test-eth-contracts:
-          filters:
-            branches:
-              ignore: /(^master$)|(.*?hotfix(.*)?)/
-      - test-creator-node:
-          filters:
-            branches:
-              ignore: /(^master$)|(.*?hotfix(.*)?)/
-      - test-discovery-provider:
-          filters:
-            branches:
-              ignore: /(^master$)|(.*?hotfix(.*)?)/
-      - test-identity-service:
-          filters:
-            branches:
-              ignore: /(^master$)|(.*?hotfix(.*)?)/
-      # - hold-test-mad-dog-e2e:
-      #     type: approval
-      #     filters:
-      #       branches:
-      #         ignore: /(^master$)|(.*?mad-?dog(.*)?)/
-      - test-mad-dog-e2e:
-          # requires:
-          #   - hold-test-mad-dog-e2e
-          filters:
-            branches:
-              ignore: /(^master$)|(.*?hotfix(.*)?)/
-
-
-
-  # push public docker images when a git tag is present
-  # tag must be in the format @audius/<service>@x.x.x
-  # else, any changes to master will be tagged `latest`
-  dockerhub-build-and-push:
+  # test and build all commits
+  test-and-build:
     jobs:
-      # libs: test, build and push
       - test-libs:
           force: true
-          filters:
-            tags:
-              only: /^@audius/libs@\d+\.\d+\.\d+/
-            branches:
-              only: /(^master$)|(.*?hotfix(.*)?)/
       - docker-build-and-push:
           name: build-libs
           repo: libs
           requires:
             - test-libs
-          filters:
-            tags:
-              only: /^@audius/libs@\d+\.\d+\.\d+/
-            branches:
-              only: /(^master$)|(.*?hotfix(.*)?)/
-      # contracts: test, build and push
+
       - test-contracts:
           force: true
-          filters:
-            tags:
-              only: /^@audius/contracts@\d+\.\d+\.\d+/
-            branches:
-              only: /(^master$)|(.*?hotfix(.*)?)/
       - docker-build-and-push:
           name: build-contracts
           repo: contracts
           requires:
             - test-contracts
-          filters:
-            tags:
-              only: /^@audius/contracts@\d+\.\d+\.\d+/
-            branches:
-              only: /(^master$)|(.*?hotfix(.*)?)/
-      # eth-contracts: test, build and push
+
       - test-eth-contracts:
-          force: true
-          filters:
-            tags:
-              only: /^@audius/eth-contracts@\d+\.\d+\.\d+/
-            branches:
-              only: /(^master$)|(.*?hotfix(.*)?)/
+          force: false
```

Comment:
why is this set to false?

Reply:
If I understand this correctly, this will run eth contract tests only when there is a diff.

Currently eth contract tests take ~ 30 minutes to run, this let's us run them only if there is a diff, so that we don't have to wait on builds.

Reply:
ah ok, I wasn't aware of how long this test suite takes. sounds good

Reply:
do we need this blob to notice the diff? 
```
          filters:
            tags:
              only: /^@audius/eth-contracts@\d+\.\d+\.\d+/
```